lib/ueberauth.ex

```diff
@@ -69,7 +69,7 @@ defmodule Ueberauth do
   In phoenix setup a pipeline:
 
       pipeline :ueberauth do
-        Ueberauth.plug "/auth"
+        plug Ueberauth, base_path: "/auth"
```

Comment:
So the defined "request route" doesn't actually do anything or router anywhere other than serving the purpose of having he pipeline invoked? If so, we should find a better option, such as plugging in the endpoint or using `forward` in the router. So instead of `plug`, you would do

```elixir
scope "/" do
  pipe_through [:browser, :ueberauth]
  forward "/auth", Ueberauth, base_path: "/auth"
end
```